biz.aQute.bndlib/src/aQute/bnd/differ/Baseline.java

```diff
@@ -285,6 +297,25 @@ public Set<Info> baseline(Tree n, Parameters nExports, Tree o, Parameters oExpor
 		return infos;
 	}
 
+	private Optional<Delta> getThreshold(Instructions packageFilters, Diff pdiff) {
+
+		var matcher = packageFilters.matcher(pdiff.getName());
+		if (matcher == null || matcher.isNegated())
+			return Optional.empty();
+
+		var attrs = packageFilters.get(matcher);
+		assert attrs != null : "guaranteed by the matcher != null";
+		var threshold = attrs.getOrDefault(Constants.DIFFPACKAGES_THRESHOLD, "MICRO")
+			.toUpperCase().trim();
```

Comment:
trim() is unnecessary. This is always properly handled by the Parameters parser.